### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by Aleksey-Bortnik**
new branch naming, with latest fix.

### CODE REVIEW

File: scss/components/_banner.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    display: flex;
+    margin-top: 45px;
+  }
+
+  &__start-button {
+    padding: 10px 30px;
+  }
+
+  &__video-link {
+    width: 179px;
```
Comment (irina-chernik) [inline scss/components/_banner.scss:37]: don't use fixed height/width

File: scss/components/_tools.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  &__card-click {
+    background-image: url("../assets/img/tools/tools-cursor.png");
+    width: 50px;
+    height: 50px;
+  }
+
+  &__card-social {
+    background-image: url("../assets/img/tools/tools-social.png");
+    width: 50px;
```
Comment (irina-chernik) [inline scss/components/_tools.scss]: use one common class for repeting styles - width and height

File: scss/components/_strategies.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.strategies-section {
+  display: flex;
+  flex-direction: column;
+
+  &__cards-container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__card{
```
Comment (irina-chernik) [inline scss/components/_strategies.scss]: add space

File: scss/components/_design.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    max-width: 486px;
+  }
+
+  &__title {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__designers-section__text {
+  font-size: 1em;
```
Comment (irina-chernik) [inline scss/components/_design.scss]: indent

**Review by irina-chernik — CHANGES_REQUESTED** [review]

**Review by irina-chernik — APPROVED** [review]

File: gulpfile.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const gulp = require("gulp");
```
Comment (angelinabylitskaya) [inline gulpfile.js]: Mixed quotes in one file. Use only one quotes style. Fix to single quotes.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "devDependencies": {
+    "gulp-sass": "^5.0.0",
+    "gulp-sourcemaps": "^3.0.0",
+    "gulp-watch": "^5.0.1",
+    "i": "^0.3.6",
+    "sass": "^1.35.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (angelinabylitskaya) [inline package.json:17]: Add some scripts to build and watch SCSS compilation. If you have scripts, you Donn't need to remember all CLI commands.

File: index.html [hunk 7]
```diff
@@ -14,239 +14,264 @@
...
+        <input class="footer__input" type="number" placeholder="Enter your phone Number">
+        <button class="footer__input-button common-btn-styles">Sing up Now</button>
       </div>
-    </footer>
-  </div>
+    </div>
+  </footer>
 </body>
 
 </html>
```
Comment (angelinabylitskaya) [inline index.html]: Empty line at the end is missing

File: scss/base/_common_classes.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+  @include text-formatter;
+}
+
+.block__container {
+  display: flex;
+  width: 100%;
+  max-width: 1126px;
+}
+
+.primary-title {
```
Comment (angelinabylitskaya) [inline scss/base/_common_classes.scss]: Well done!
You can separate layout classes and typography classes in corresponding files.

**Pull request merged**